include/ocpp1_6/smart_charging.hpp

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
+
+#ifndef OCPP1_6_SMART_CHARGING_HPP
+#define OCPP1_6_SMART_CHARGING_HPP
+
+#include <limits>
+
+#include <ocpp1_6/connector.hpp>
+#include <ocpp1_6/database_handler.hpp>
+#include <ocpp1_6/ocpp_types.hpp>
+#include <ocpp1_6/transaction.hpp>
+
+namespace ocpp1_6 {
+
+static const int LOW_VOLTAGE = 230;
+static const int DEFAULT_AND_MAX_NUMBER_PHASES = 3;
+static const int HOURS_PER_DAY = 24;
+static const int SECONDS_PER_HOUR = 3600;
+static const int SECONDS_PER_DAY = 86400;
+static const int DAYS_PER_WEEK = 7;
```

Comment:
Using `static` here is not a good idea in general, because it will create an instance in every compilation unit (cpp file), that is including this header file.